about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Body Bulding</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+  <script src="https://kit.fontawesome.com/c6f38472f5.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header>
+    <div class="wrapper flex">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My Page</a>
+      <a href="#">Logout</a>
+    </div>
+  </header>
+
+  <i class="fa-sharp fa-solid fa-bars menu"></i>
+  <div class="navigation">
+    <i class="fa-solid fa-xmark xmark"></i>
+    <nav class="wrapper">
+      <div class="nav-top">
+        <a href="index.html">
+          <img class="logo" src="assets/images/logo2.png" alt="logo">
+          <p class="max-font-26"><span>SP Fitness</span><br>center 2023</p>
+        </a>
+      </div>
+      <ul class="nav-links flex column">
+        <li><a class="nav-link max-font-26" href="about.html">About</a></li>
+        <li><a class="nav-link max-font-26" href="#">Program</a></li>
+        <li><a class="nav-link max-font-26" href="#">Join</a></li>
+        <li><a class="nav-link max-font-26" href="#">Sponsor</a></li>
+        <li><a class="nav-link max-font-26 " href="#">News</a></li>
+      </ul>
+      <div class="nav-bottom flex">
+        <button class="max-font-26" type="button">SPFC Story</button>
+      </div>
+    </nav>
+  </div>
+  <main>
+    <section id="headline" class="section">
+      <div class="greeting">
+        <p>"Hello! Champion"</p>
+      </div>
+      <div class="wrapper">
+        <h1>Super Power Fitness Center 2023</h1>
```

Comment:
### Your About Page Title

![image](https://user-images.githubusercontent.com/93039359/235136989-6bd1e502-fa0a-46e3-8588-a3d4ee303cda.png)

### Design About Page Title

![image](https://user-images.githubusercontent.com/93039359/235137119-a329d4c8-f185-457d-9f13-a616265256a9.png)

- [ ] As suggested by the previous reviewer, kindly reduce the width of this `h1` to make your heading width similar to the design title to make your project attractive :+1: